localization/english/modifiers/wc_activity_feast_modifiers_l_english.yml

```diff
@@ -0,0 +1,8 @@
+﻿l_english:
+ feast_cavalry_insight_modifier:0 "Cavalry and Aerial Insight"
+ feast_cavalry_insight_modifier_desc:1 "Plotting out theoretical strategies has given this character ideas for how to more efficiently use their cavalry and aerial."
+ feast_heavy_infantry_insight_modifier:0 "Heavy Infantry and Giants Insight"
+ feast_heavy_infantry_insight_modifier_desc:1 "Plotting out theoretical strategies has given this character ideas for how to more efficiently use their giants and heavy infantry."
+ feast_archers_insight_modifier:0 "Archers and Casters Insight"
+ feast_archers_insight_modifier_desc:1 "Plotting out theoretical strategies has given this character ideas for how to more efficiently use their archers and casters."
```

Comment:
Guess what, it doesn't work.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/105343977-c2ea8380-5bfb-11eb-962e-6a26512f2e7f.png)

</details>